### PR TITLE
fix: invalid interface MTU values

### PR DESCRIPTION
**Pull request by jajeffries**
This pull request improves the robustness of the `InterfaceMapper` by adding validation for MTU values and extends the test suite to cover edge cases for MTU mapping. The validation ensures that MTU values less than or equal to 0, negative values, or empty values are handled gracefully, resulting in a `nil` MTU.

### Validation Enhancements:
* **MTU Validation in `InterfaceMapper`**: Added a check to skip MTU values that are less than or equal to 0, logging a warning when such values are encountered. (`snmp-discovery/mapping/mappers.go`, [snmp-discovery/mapping/mappers.goR289-R292](diffhunk://#diff-480b988f8d1afe0ad0f2780a5b6bfa716acc21e684a110065226f573af92925eR289-R292))

### Test Coverage Improvements:
* **New Test Cases for MTU Handling**: Added test cases to verify that MTU values of 0, negative values, and empty values result in a `nil` MTU in the mapped entity. (`snmp-discovery/mapping/mappers_test.go`, [snmp-discovery/mapping/mappers_test.goR600-R761](diffhunk://#diff-683ef3f3451d82e76a861bc708a56f8b84a3cdaec883501f378724e5c3b93130R600-R761))

### CODE REVIEW

File: snmp-discovery/mapping/mappers_test.go [hunk 1]
```diff
@@ -597,6 +597,168 @@ func TestInterfaceMapper_Map(t *testing.T) {
...
+						Entity: "interface",
+						Field:  "name",
+					},
+					{
+						OID:    "1.3.6.1.2.1.2.2.1.4",
+						Entity: "interface",
+						Field:  "mtu",
+					},
+				},
+			},
```
[the remark this answers is not in the document]
Comment (jajeffries) [reply snmp-discovery/mapping/mappers_test.go:646]: Not for now

**Review by leoparente — APPROVED** [review]
